### PR TITLE
KS-SK998-htaccess-wordpress-translations

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations and links done
gitpod links => gitpod ko, currently he doesn't purpose to open the render. This after two runs 
render =>gitpod ko, currently he doesn't purpose to open the render. This after two runs 

need proof for DE, EN, latin and PL versions.

### CODE REVIEW

**Comment by Kazarus4** [comment]
render and links done and checked with gitpod

File: pages/web/hosting/htaccess_how_to_protect_wordpress/guide.en-asia.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
+title: "Tutorial - Using the htaccess file with WordPress"
+slug: use-htaccess-with-wordpress
+excerpt: "Find out how to secure your WordPress blog with one or more htaccess files"
+section: 'Tutorials'
+order:
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_how_to_protect_wordpress/guide.en-asia.md]: please add an order, it should not be empty

File: pages/web/hosting/htaccess_how_to_protect_wordpress/guide.es-es.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+### ¿Qué precauciones deben tomarse?
+
+Una mala configuración de su fichero **.htaccess** puede generar errores en su servidor (como un error 500 : *Internal Server Error*) o incluso hacer que su servicio deje de estar disponible, incluso para usted. OVH se encarga de realizar copias de seguridad sistemáticas de las versiones de sus archivos funcionales para poder volver a un estado anterior en caso de fallo tras un cambio.
+
+De la misma manera, si no está acostumbrado a manipular este tipo de archivo, pruebe cada elemento que edite. Así evitará perder tiempo en la recuperación de la línea o líneas causantes del fallo de funcionamiento de su servidor web. Un error de configuración o un simple error de escritura pueden comprometer la configuración y el funcionamiento del servidor web.
+
+### ¿Qué herramientas utilizar?
+
+un cliente FTP para recuperar sus archivos (FileZilla, Cyberduck).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_how_to_protect_wordpress/guide.es-es.md]:
```suggestion
- un cliente FTP para recuperar sus archivos (FileZilla, Cyberduck).
```

File: pages/web/hosting/htaccess_how_to_protect_wordpress/guide.es-es.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+- **RewriteRule**: esta sintaxis se escribe en el esquema de `RewriteRule Modelo Sustitution`. La escritura puede aparecer varias veces en el archivo **.htaccess** (es el caso en el archivo predeterminado que se encuentra en la raíz de la instalación de su WordPress). El orden de escritura en el archivo es primordial, tenga cuidado con el orden en que escriba sus reglas.
+- **RewriteBase**: especifica que la raíz del sitio sea `/`.
+- **RewriteCond** : son condiciones previas para la regla que sigue directamente. En nuestro caso, la primera condición excluye las URL que contienen una ruta a un archivo real, mientras que la segunda excluye las subcarpetas.
+
+### ¿Qué puedo añadir a un archivo **.htaccess** con WordPress?
+
+Existen varias formas de definir y modificar los parámetros que cambiarán el comportamiento del servidor (aunque existen limitaciones en función del alojamiento):
+
+- modificar los archivos de configuración del servidor;
+Añadir o modificar directrices en el archivo de configuración **wp-config.php** en la raíz del sitio web.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_how_to_protect_wordpress/guide.es-es.md]:
```suggestion
- Añadir o modificar directrices en el archivo de configuración **wp-config.php** en la raíz del sitio web.
```

File: pages/web/hosting/htaccess_how_to_protect_wordpress/guide.it-it.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+```
+
+### Informazioni importanti da ricordare
+
+- Salva una versione funzionale del tuo file **.htaccess** prima di effettuare qualsiasi operazione.
+- Se le modifiche apportate provocano un errore, sostituisci (tramite il tuo client FTP) il file **.htaccess** online con la versione precedente.
+- Puoi gestire alcuni parametri nel tuo file **wp-config.php**
+- I file **.htaccess** sono particolarmente efficaci per la gestione di URL, reindirizzamenti e la sicurezza del tuo sito Web.
+
+## Spingiti oltre <a name="go-further"></a>
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_how_to_protect_wordpress/guide.it-it.md]: Please check translation here, does not look like the usual Go Further one

Comment (Kazarus4) [reply pages/web/hosting/htaccess_how_to_protect_wordpress/guide.it-it.md]: DONE

File: pages/web/hosting/htaccess_how_to_protect_wordpress/guide.en-asia.md (hunk 1 above)
Comment (Kazarus4) [reply pages/web/hosting/htaccess_how_to_protect_wordpress/guide.en-asia.md]: done

**Review by bs-ovh — APPROVED** [review]

File: pages/web/hosting/htaccess_how_to_protect_wordpress/guide.fr-fr.md [hunk 5]
```diff
@@ -24,7 +24,7 @@ Ce tutoriel vous explique comment configurer certaines fonctionnalités de votre
 ## Prérequis
 
 - Disposer d'un [hébergement web](https://www.ovhcloud.com/fr/web-hosting/) et avoir installé WordPress.
-- Être en capacité d'utiliser un client FTP tel que [FileZilla](https://filezilla-project.org/). Vous pouvez consulter notre guide « [Utiliser FileZilla pour récupérer et déposer vos données](https://docs.ovh.com/fr/dedicated/deposer-et-recuperer-donnees-via-sftp/#utiliser-filezilla-pour-recuperer-et-deposer-vos-donnees) ».
+- Être en capacité d'utiliser un client FTP tel que [FileZilla](https://filezilla-project.org/). Vous pouvez consulter notre tutoriel « [Filezilla](https://docs.ovh.com/fr/hosting/mutualise-guide-utilisation-filezilla/) ».
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_how_to_protect_wordpress/guide.fr-fr.md]:
```suggestion
- Être en capacité d'utiliser un client FTP tel que [FileZilla](https://filezilla-project.org/). Vous pouvez consulter notre [tutoriel sur l'utilisation de FileZilla](https://docs.ovh.com/fr/hosting/mutualise-guide-utilisation-filezilla/).
```

File: pages/web/hosting/htaccess_how_to_protect_wordpress/guide.fr-ca.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+> 
+> Nous mettons à votre disposition ce tutoriel afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [prestataire spécialisé](https://partner.ovhcloud.com/fr-ca/) ou [l'éditeur du CMS WordPress](https://wordpress.com/fr/support/){.external} si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [« Aller plus loin »](#go-further) de ce tutoriel.
+>
+
+**Découvrez comment sécuriser votre WordPress avec un ou plusieurs fichiers htaccess.**
+
+## Prérequis
+
+- Disposer d'un [hébergement web](https://www.ovhcloud.com/fr-ca/web-hosting/) et avoir installé WordPress.
+- Être en capacité d'utiliser un client FTP tel que [FileZilla](https://filezilla-project.org/). Vous pouvez consulter notre tutoriel « [Filezilla](https://docs.ovh.com/fr/hosting/mutualise-guide-utilisation-filezilla/) ».
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/htaccess_how_to_protect_wordpress/guide.fr-ca.md]:
```suggestion
- Être en capacité d'utiliser un client FTP tel que [FileZilla](https://filezilla-project.org/). Vous pouvez consulter notre [tutoriel sur l'utilisation de FileZilla](https://docs.ovh.com/fr/hosting/mutualise-guide-utilisation-filezilla/).
```

**Pull request merged**